ui/app/templates/components/console/log-error-with-html.hbs

```diff
@@ -1,5 +1,5 @@
 {{! template-lint-disable no-triple-curlies}}
 <div class="console-ui-alert has-text-danger">
-  <Icon @glyph="cancel-circle-fill" aria-hidden="true" />
+  <Icon @name="x-circle-fill" />
```

Comment:
I think we should keep `aria-hidden` wherever we had it in the dom before -- quite a few places where this was removed

Reply:
I removed these on the instances where the `name` was from the flight set since they add `aria-hidden="true"` to the underlying svg by default.
![image](https://user-images.githubusercontent.com/24611656/140093891-c32cf5fd-dd4d-4cd6-95e1-1937c806e047.png)